src/bindings/python/tests/test_runtime/test_sync_infer_request.py

```diff
@@ -395,16 +395,16 @@ def test_get_compiled_model(device):
                          [np.float32,
                           np.int32,
                           np.float16])
-@pytest.mark.parametrize("mode", ["set_init_memory_state", "reset_memory_state", "normal"])
+@pytest.mark.parametrize("mode", ["set_init_memory_state", "reset_memory_state", "normal", "reset_via_infer_request"])
 @pytest.mark.parametrize("input_shape", [[10], [10, 10], [10, 10, 10], [2, 10, 10, 10]])
 @pytest.mark.skipif(
     os.environ.get("TEST_DEVICE", "CPU") != "CPU",
```

Comment:
@jiwaszki GPU also supports stateful models and we should not skip it here

Reply:
@itikhono can you add this to your PR?
```suggestion
    os.environ.get("TEST_DEVICE", "CPU") not in ["CPU", "GPU],
```

Reply:
done